clients/src/main/java/org/apache/kafka/clients/consumer/internals/ListOffsetsRequestManager.java

```diff
@@ -109,38 +109,42 @@ public NetworkClientDelegate.PollResult poll(final long currentTimeMs) {
         return pollResult;
     }
 
+
     /**
      * Retrieve offsets for the given partitions and timestamp.
      *
-     * @param partitions        Partitions to get offsets for
-     * @param timestamp         Target time to look offsets for
-     * @param requireTimestamps True if this should fail with an UnsupportedVersionException if
-     *                          the broker does not support fetching precise timestamps for offsets
-     * @return Future containing the map of offsets retrieved for each partition. The future will
-     * complete when the responses for the requests are received and processed following a call
-     * to {@link #poll(long)}
+     * @param timestampsToSearch Partitions and target timestamps to get offsets for
```

Comment:
Moderate nit: please add the missing Javadoc for `requireTimestamps`:

```suggestion
     * @param timestampsToSearch Partitions and target timestamps to get offsets for
     * @param requireTimestamps  <FILL IN...>
```